backend/typescript/services/implementations/storageService.ts

```diff
@@ -0,0 +1,89 @@
+import { storage } from "firebase-admin";
+
+import IFileStorageService from "../interfaces/storageService";
+import logger from "../../utilities/logger";
+
+const Logger = logger(__filename);
+
+class FileStorageService implements IFileStorageService {
+  bucket: any;
+
+  constructor(bucketName: string) {
+    this.bucket = storage().bucket(`gs://${bucketName}}`);
+  }
```

Comment:
Made `bucketName` a parameter to allow using multiple `FileStorageService` instances with different bucket names. Initializing `firebase-admin` with a bucket name in `server.ts` is not required.